Fix libunwind build for the following two reason

aokblast opened the pull request:
1. Libunwind has hand-coded asm code and should be enabled by project or it will not compile assembly part

2. We should install libunwind header or we are unable to use it.

KKoovalsky commented:
Thanks for contributing!

All good, but could you turn the LIBUNWIND_INSTALL_HEADERS into an option?

aokblast committed:
Fix libunwind build for the following two reason

1. Libunwind has hand-coded asm code and should
   be enabled by project or it will not compile
   assembly part

2. We should install libunwind header or we are
   unable to use it.

Sponsered by: CloudMosa Inc.

aokblast commented:
Fix it!

aokblast commented:
Hello, do you have any update about it?

KKoovalsky commented:
I need to test it. I'm short on time recently, so I'll merge and trust you it works. Then I will test. ;)

aokblast commented:
Oops. Sorry about that. I am not malicious on this. I am just wondering if you forget this. Sorry.
